front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/SurveyResultsWidget/SurveyResults.tsx

```diff
@@ -128,27 +116,4 @@ const SurveyResults = ({
   );
 };
 
-const SurveyResultsWrapper = (props: Props) => {
-  const { slug, phaseNumber } = useParams();
-  const { data: project } = useProjectBySlug(slug);
-  const { data: phases } = usePhases(project?.data.id);
-
-  const selectedPhase = useMemo(() => {
-    if (!phases) return;
-
-    // if a phase parameter was provided, and it is valid, we set that as phase.
-    // otherwise, use the most logical phase
-    if (isValidPhase(phaseNumber, phases.data)) {
-      const phaseIndex = Number(phaseNumber) - 1;
-      return phases.data[phaseIndex];
-    }
-
-    return getLatestRelevantPhase(phases.data);
-  }, [phaseNumber, phases]);
-
-  const reportId = selectedPhase?.relationships.report?.data?.id;
-
-  return <SurveyResults {...props} reportId={reportId} />;
-};
-
-export default SurveyResultsWrapper;
```

Comment:
@luucvanderzee Should I do the same here? https://github.com/CitizenLabDotCo/citizenlab/blob/8808fcd4dc90f3144d17a2fa9bdf628f7f46838b/front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/ChartWidgets/ReactionsByTimeWidget/ReactionsByTimeCard.tsx#L73-L94

Reply:
Could you clarify?

Reply:
@luucvanderzee Indeed not very clear. So, in my code, to get `reportId`, instead of using a custom wrapper with multiple hooks and some logic, I did
```ts
const { reportId } = useReportContext();
```
Should I do the same instead of `ReactionsByTimeWrapper`?

Reply:
Yes that's perfect, `useReportContext` indeed exists to make it easy to get these values. You can use it in any widget or any other component that is inside of the `ReportContextProvider`

Reply:
Done in this commit https://github.com/CitizenLabDotCo/citizenlab/pull/6605/commits/f4566837891059e417fe9443230b8fbb096c158b (in the current PR).